web/satellite/src/components/common/HeaderedInput.vue

```diff
@@ -14,131 +14,159 @@
 			<h4>Optional</h4>
 		</div>
 		<textarea
-				v-if="isMultiline"
-				:id="this.$props.label"
-				:placeholder="this.$props.placeholder"
-				v-model="value"
-				:style="style"
-				:rows="5"
-				:cols="40"
-				wrap="hard"
-				@input="onInput" >
+            v-if="isMultiline"
```

Comment:
Right now, we don't have any multiline-prefilled components.
Just in case it might appear in future I will duplicate this functionality to <textArea>